### DISCUSSION TITLE
Text above matrix

### DISCUSSION THREAD

**Discussion by sergiokapone** [lvjr/tabularray#248] [Q&A]
Hi. Thank you for package. It is really revolution in talbe typesetting!!!

I want to ask a question about problem, mentioned in [SE](https://tex.stackexchange.com/questions/42666/text-above-matrices)
How can I add text above the matrices with +bmatrix, +pmatrix etc?

I did not find this in the manual (maybe I was looking badly), it seems to me that it is poorly described there, maybe I need an example.

**Comment by lvjr** [comment]
**Note: I have moved this question from "GitHub Issues" to here ("GitHub Discussions").**

Without a minimal example, it is hard to say whether or not it is possible to do it with `tabularray` package. But I think the solutions in the TeX.SE page which you provided can be used for `+bmatrix` environment. Just replace `bmatrix` with `+bmatrix`.
